guides/common/modules/proc_copying-a-content-view.adoc

```diff
@@ -0,0 +1,29 @@
+[id="Copying_a_Content_View_{context}"]
+= Copying a content view
+
+Use this procedure to copy a content view.
+You can copy a content view in the {ProjectWebUI} or you can use the Hammer CLI to copy an existing content view into a new content view.
```

Comment:
Can you add an anchor to the CLI procedure below starting with `cli-` and link to it? That's the standard way we use elsewhere.